pset-iterators.js

```diff
@@ -115,6 +137,18 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+	const tookOffice = parseInt(president.took_office)
+	const leftOffice = parseInt(president.left_office)
+	if(tookOffice >= 1850 && leftOffice <= 1900) {
+		return president.president
+	}
+}).map((president) => {
+    return president.president
+})
+
+console.log(`Presidents who held office between the years 1850 and 1900 are: ${presidentsBetween1850and1900}.`)
+
```

Comment:
@cswebdesign Nice work here in this solution. Utilizing the filter method to only return the presidents who served between 1850 and 1900 was a great approach to this problem. One suggestion for this problem you don't need the chained map method. Since the filter method will return an array, chaining the map method would give you the individual presidents names but this problem required you only to return the array of each president's object who served between 1850 and 1900.

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L124)